Articles/Blog/2023-04-developer-surveys.md

```diff
@@ -8,114 +8,126 @@
 Even when we know there is a huge problem that needs attention, we might not have the people power to do it.
-So we decided that we would spread the overall messages of the survey to the teams or leaders that might be empowered to do something.
+So we decided that we would spread the overall message of the survey to these teams so they might be empowered to do something.
 As an example, one strong piece of insight we got (that is public) is that developers really like VSCode.
-It was very easy for me to see this [this announcement from GitLab](https://devclass.com/2022/12/20/gitlab-goes-public-with-web-ide-beta-based-on-visual-studio-code/) and realize that getting this integration enabled for our private GitLab instances would be hugely valued.
-It was then relatively easy to approach one of the leads of the team that maintains GitLab, express the interest, and ask if this could be provided.
+It was very easy for me to see this and pair it with [this announcement from GitLab](https://sdtimes.com/software-development/gitlab-announces-beta-of-its-new-web-ide/) and approach one of the leads of the team that maintains GitLab at LLNL and ask if this could be provided.
 It was also easy to see that many folks were asking for a piece of software to be updated, but then to go to the team responsible and find out that they had good reasons to not be able to do it yet.
 
-This leads to my greatest worry about the survey - that we might put it out, get great insights, but not be empowered to take action on many of them.
+*Q:* Is there anything you would do differently?
+
+*Vanessa*: One of my greatest worries about the survey is that we might put it out, get great insights, but not be empowered to take action on many of them.
 I'm hopeful that if we identify many problems, if we can minimally target two to three of them a year to work on for that year, we can slowly make progress toward better developer experience and happiness.
 
-### Conclusion
+*Q*: Any closing thoughts?
 
-You can read our first public set of highlights at [the RADIUSS news](https://software.llnl.gov/radiuss/2023/02/08/first-developer-survey/)
+*Vanessa*: You can read our first public set of highlights at [the RADIUSS news](https://software.llnl.gov/radiuss/2023/02/08/first-developer-survey/)
 blog.
 We hope that other academic institutions and/or national labs take interest in the happiness and productivity of their developers, and consider contributing to this space.
 An important point to remember is that while we may not be able to respond to every desire or gripe, the fact that we are asking, period, means that we are trying our best, and our hearts are in the right place.
 Even large, meaningful changes can start with small steps, and this kind of effort is exactly that.
 Perhaps if we could identify shared gripes, the combined forces of multiple institutions would have enough people power to work on them?
```

Comment:
This sounds better as an ending (instead of ending on the question).
```suggestion
Perhaps if we could identify shared gripes, the combined forces of multiple institutions would have enough people power to work on them?
This is something I would be excited about.
```